File: codeforces/low_difficulty/difficulty_800/92.vanya_and_cubes_[492A]/vanya_and_cubes.py
# Author: José Rodolfo (jric2002)
n = int(input())
h = 0
i = 1
while (n >= 0):
  n = n - int((i * (i + 1)) / 2)
  if (n >= 0):
    h += 1
  i += 1
print(h)

File: codeforces/low_difficulty/difficulty_800/92.vanya_and_cubes_[492A]/vanya_and_cubes.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  short int n;
  unsigned short int h, i;
  cin >> n;
  h = 0;
  for (unsigned short int i = 1; n >= 0; i++) {
    n = n - ((i * (i + 1) / 2));
    if (n >= 0) {
      h++;
    }
  }
  cout << h << endl;
  return 0;
}